forge/src/main/java/org/openjdk/skara/forge/PullRequestUtils.java

```diff
@@ -163,6 +163,11 @@ public static Repository materialize(HostedRepositoryPool hostedRepositoryPool,
         return localRepo;
     }
 
+    public static boolean isAncestorOfTarget(Repository localRepo, Hash hash) throws IOException {
```

Comment:
I put this in the utils class because it needs the "prutils_tagetref" reference, which is currently only used in this class.